trustregistry/tests/test_actor.py

```diff
@@ -12,6 +12,34 @@
     "didcomm_invitation": "string",
     "did": "did:key:string",
 }
+dup_name_actor = {
+    "id": "darth-vader2",
+    "name": "Darth Vader",
+    "roles": ["issuer", "verifier"],
+    "didcomm_invitation": "string2",
+    "did": "did:key:string2",
+}
+dup_did_actor = {
+    "id": "darth-vader3",
+    "name": "Darth Vader3",
+    "roles": ["issuer", "verifier"],
+    "didcomm_invitation": "string3",
+    "did": "did:key:string",
+}
+dup_didcomm_actor = {
+    "id": "darth-vader4",
+    "name": "Darth Vader4",
+    "roles": ["issuer", "verifier"],
+    "didcomm_invitation": "string",
+    "did": "did:key:string4",
+}
+dup_id_actor = {
+    "id": "darth-vader",
+    "name": "Darth Vader5",
+    "roles": ["issuer", "verifier"],
+    "didcomm_invitation": "string5",
+    "did": "did:key:string5",
+}
```

Comment:
It feels like there should be a way to do this without the duplicate definitions ...
Like, you can define a def generate_actor(), where it returns a dict with keys: id, name, didcomm_invitation, did; where they are all randomly generated using our helper method (from app.tests.util.string import random_string)

So you can say dup_did_actor = generate_actor()
dup_did_actor["did"] = predefined_string

This way it's less duplication and more clear which key is being recycled